### ISSUE TITLE
Missing geometry on scene-viewer

### ISSUE THREAD

**Issue by MajorMeerkatThe3rd** [google/model-viewer#3849]
### Description

When loading [this model](
https://shapediverviewer.s3.amazonaws.com/v3/gltf/example.gltf) in the scene-viewer,  parts of the geometry is missing. It should be three planes and three cubes, but only the three planes are visible. The GLTF seems to be in order, no issues when I check the model details on https://gltf-viewer.donmccurdy.com/. This does not happen in WebXR or when the scene-viewer was created via the model-viewer (with this page at least https://modelviewer.dev/editor/).

Is there a difference in how I load the scene-viewer compared to the example below? I couldn't find anything.

#### Live Demo

A very easy CodeSandBox to reproduce the issue:
https://codesandbox.io/s/6rrwto

The model (three cubes and three planes):
https://shapediverviewer.s3.amazonaws.com/v3/gltf/example.gltf

It works when I load the same model on the [model-viewer editor](https://modelviewer.dev/editor/) and select "Default AR Mode to Scene Viewer". I looked at the code of the model-viewer and how the scene-viewer is loaded there and I could not find a difference.

### Version
scene-viewer 1.0 / 1.1

### Browser Affected
-

### OS
- [ X ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [ X ] SceneViewer
- [ ] QuickLook

**Comment by MajorMeerkatThe3rd** [comment]
Hey @elalish ,

sure, so [here](https://shapediverviewer.s3.amazonaws.com/v3/images/working.jpg) you can see how it should look like and how it does look like when I start the scene-viewer via the model-viewer.

And [here](https://shapediverviewer.s3.amazonaws.com/v3/images/not_working.jpg) is how it looks like when loaded directly in the scene-viewer as I show in the CodeSandBox.

Cheers

**Comment by ukito-pl** [comment]
Hi, I am also experiencing this issue with missing geometry. 
Problem appears if more than one node has the same name! According to the GLTF spec, property "name" should not be interpreted as unique value, it is not even required. If I remove the name properties from nodes, everything works fine, but if two nodes have the same name it all breaks down. It really is a strange and very troublesome bug! Modifing all exsiting models to work around such a simple issue is a real pain. 

Can we expect any fix soon?

**Comment by jukibom** [comment]
Yep, verified here too - doesn't happen on a colleagues' old phone which hasn't updated SceneViewer (/Google app).

This model doesn't work, it breaks in weird ways (some models become transparent, others fail to render entirely)
[Talia 1G.zip](https://github.com/google/model-viewer/files/9724081/Talia.1G.zip)


This model works (same but manually edited the nodes with gibberish)
[Talia1GTest.zip](https://github.com/google/model-viewer/files/9724074/Talia1GTest.zip)

Switching over to WebXR as a reasonable stopgap for now but SceneViewer still provides the best experience, sadly (especially on older devices)

**Comment by elalish** [comment]
It's been reported and the regression identified. A fix should roll out over the next week or two.

**Comment by goout** [comment]
a same issue, hope on soonest fix

**Comment by kremisi** [comment]
I have a similar issue, hope it'll be fixed soon, I'm following this git to receive good news.
